### DISCUSSION TITLE
Some of my plots has disappeared

### DISCUSSION THREAD

**Discussion by Epemba1978** [Chia-Network/chia-blockchain#8150] [Support]
I move my PC to a new case and added some new HDD. My plots dropped from 72 to 43, one of my 4TB HDD says 7 plots but the HDD is full. What happened?

**Comment by BrandtH22** [comment]
closing as this is a duplicate discussion.

Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
